### PR TITLE
Update press kit with new branding.

### PR DESCRIPTION
**Pull request by brendastorer**
presskit

Updated presskit with sweet new fonts and thoughtbot red! Other changes:

- Removed the "logo" folder which wasn't linked to anywhere, and was the same thing as the `thoughtbot-assets-pack.zip`

- Also updated `thoughtbot-assets-pack.zip` and it now includes the svgs that were once in the "logo" folder

- thoughtbot now is a solid color, all red. No more "bot" in gray

- Copy updates around changes where needed, like with the wordmark and the fonts

I think we could take this further with some design updates and refactoring, but wanted to get the basics in that could be used immediately.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -180,18 +180,19 @@ <h4 class="color-notes">
         <section id="typography">
           <h2>Typography</h2>
           <p>
-            We primarily use the Calluna Sans font family for headings: straightforward but full of personality. We use freight text pro with copy for its elegance and excellent legibility.
+            We primarily use the Calibre font family. It's straightforward, but full of personality. We typically use regular weight for body copy and medium weight for headings.
```
Comment (ohaicristina) [inline index.html]: Is Calibre also straightforward but full of personality? Or are there other descriptors that make more sense here

File: index.html [hunk 2]
```diff
@@ -180,18 +180,19 @@ <h4 class="color-notes">
...
-          <div class="body-typeface">
-            <h3 class="typeface-name">Freight Text Pro</h3>
-            <h3 class="uppercase">abcdefghijklmnopqrstuvwxyz</h3>
-            <h3 class="lowercase">abcdefghijklmnopqrstuvwxyz</h3>
+          <div class="medium-typeface">
+            <h3 class="typeface-name">Calibre Medium</h3>
+            <p class="uppercase">abcdefghijklmnopqrstuvwxyz</p>
+            <p class="lowercase">abcdefghijklmnopqrstuvwxyz</p>
           </div>
+
```
Comment (ohaicristina) [inline index.html]: ✂️ ?

**Comment by ohaicristina** [comment]
Some minor code comments! But looking real good.

Is the sidebar active class not updating as you scroll? Or is the .gif being weird? It would be nice to have that work before we make it live.

File: stylesheets/sass/_main.scss [hunk 3]
```diff
@@ -64,20 +50,10 @@ img {
...
+.button--github {
+  @include button($medium-gray);
   margin-top: 20px;
-  padding: 5px 10px 10px;
-
-  &:hover {
-    color: $medium-gray;
-    opacity: 0.8;
-  }
+  padding: 2px 15px 10px 10px;
```
Comment (smharley) [inline stylesheets/sass/_main.scss]: could this be moved to the `button` mixin? looks like it's being used on both types of buttons

File: stylesheets/sass/base/_fonts.scss [hunk 4]
```diff
@@ -0,0 +1,19 @@
+@include font-face("Calibre", "../fonts/CalibreWeb-Regular") {
+  font-style: normal;
+  font-weight: 400;
```
Comment (smharley) [inline stylesheets/sass/base/_fonts.scss]: can we use variables here? `$medium-font-weight` seems to be defined, can we define (some of) the variables before the fonts?

File: stylesheets/sass/base/_typography.scss [hunk 5]
```diff
@@ -15,14 +15,14 @@ h5,
...
-  font-weight: 300;
+  font-weight: $medium-font-weight;
   line-height: $heading-line-height;
   margin: 0 0 1.2em;
   text-rendering: optimizeLegibility;
 }
 
 h1 {
-  font-size: $base-font-size * 4; // 16 * 4 = 65px
+  font-size: $base-font-size * 3; // 16 * 4 = 65px
```
Comment (smharley) [inline stylesheets/sass/base/_typography.scss]: is the comment here still relevant?

File: stylesheets/base.css [hunk 6]
```diff
@@ -1,31 +1,50 @@
 @import url(../stylesheets/normalize.css);
-button, [type='button'], [type='reset'], [type='submit'] {
+@font-face {
+  font-family: "Calibre";
+  src: url("../fonts/CalibreWeb-Regular.woff2") format("woff2"), url("../fonts/CalibreWeb-Regular.woff") format("woff"), url("../fonts/CalibreWeb-Regular.ttf") format("truetype");
```
Comment (smharley) [inline stylesheets/base.css]: you have `.ttf` here, but don't include any `.ttf` files, just `.eot`

File: index.html (hunk 1 above)
Comment (brendastorer) [inline index.html]: I am quite insecure about my copywriting ability and would love suggestions if anyone is game. Also, just kind of made that up and not sure why we chose Calibre. Maybe @kylefiedler or @ColeTownsend could help with some good adjectives?

File: stylesheets/sass/base/_variables.scss [hunk 7]
```diff
@@ -1,13 +1,21 @@
+// Bourbon settings
+$bourbon: (
+  "global-font-file-formats": ("eot", "woff2", "woff"),
```
Comment (tysongach) [inline stylesheets/sass/base/_variables.scss]: We can remove `eot` fonts, as support for WOFF and WOFF2 matches what we need:

http://caniuse.com/#feat=woff
http://caniuse.com/#feat=woff2

**Comment by brendastorer** [comment]
Thanks @ohaicristina @tysongach @smharley for your feedback! I made the code adjustments you suggested, but have left the larger comments for future branches and improvements, in order to get this two month old PR merged on in.

**Pull request merged**